Wurstpack/wurstscript/lib/systems/Knockback3.wurst

```diff
@@ -0,0 +1,217 @@
+package Knockback3
+
+import Terrain
+import MapBounds
+import LinkedListModule
+import HashMap
+
```

Comment:
@Frotty , The `destroyDestructableSpeedThreshold` and `destroyDestructableHeightThreshold` values are `static var`, which means they are accessible in imported packages.

When you say make them configurable, do you mean, change them to use `@configurable` instead? If so, am I correct to say that this is just for consistency?